draft-ietf-quic-qpack.md

```diff
@@ -586,8 +588,10 @@ d = count of entries dropped
 ### Prefixed Integers
 
 The prefixed integer from Section 5.1 of [RFC7541] is used heavily throughout
-this document.  The format from [RFC7541] is used unmodified.  QPACK
-implementations MUST be able to decode integers up to 62 bits long.
+this document.  The format from [RFC7541] is used unmodified.  Note, however,
+that QPACK uses some prefix sizes not actually used in HPACK.
```

Comment:
I was actually tripped by this when re-using the HPACK implementation, because that had DCHECKs on prefix length that restricted it to values that occur in HPACK.  And of course there was no test coverage for other prefixes.  In particular, (3+) and (8+) occur in the current QPACK draft but not in RFC7541.